style.css

```diff
@@ -0,0 +1,594 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+}
+
+.firstNav {
+  background-color: rgb(76, 37, 38);
+}
+
+.navList {
+  display: flex;
+  list-style: none;
+}
+
+.firstNav .navList {
+  justify-content: flex-end;
+  gap: 2.188rem;
+  padding: 1% 17%;
+}
+
+.nav-content {
+  text-decoration: none;
+}
+
+.firstNav .nav-content {
+  color: rgb(248, 100, 100);
+}
+
+.secondNav {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.menuIcon {
+  display: none;
+}
+
+#close {
+  display: none;
+}
+
+.secondNav .navList {
+  flex-wrap: wrap;
+  font-size: 1.125rem;
+  gap: 1.563rem;
+}
+
+.logo img {
+  width: 200px;
+  height: 70px;
+}
+
+.secondNav .nav-content {
+  color: black;
+}
+
+.secondNav .navBtn {
+  border: 4px solid rgb(59, 45, 102);
+  color: rgb(60, 64, 103);
+  padding: 15%;
+}
+
+.firstSection {
+  height: 100vh;
+  background-image: url(./assets/bg_img.jpg);
+}
+
+.inner-div {
+  text-align: center;
+  padding-block: 6%;
+  padding-inline: 4%;
+}
+
+.heading {
+  color: orangered;
+  font-size: 1.875rem;
+}
+
+.sub-title {
+  line-height: 1.563rem;
+  font-size: 30px;
+  padding: 2%;
+  color: orangered;
+}
+
+.head-title {
+  color: white;
+  font-weight: bold;
+  font-size: 70px;
+  background-image: url(./assets/orangeImg.jpg);
+  letter-spacing: 5px;
+}

```

Comment:
- [ ] As specified in the project requirements, You can customize the content of the design but maintain the layout of the design. Currently, the layout of your header section design match with that of the design so kindly makes to style properly to match it. 
- [ ] Also the background color of your topnav doesn't match the given design so kindly make sure to change it so it matches.
**Note: Please maintain the colors of the design as specified in the project requirement.**
<img width="1088" alt="Screenshot 2023-06-01 at 11 15 26" src="https://github.com/Adesina-Abiodun/Capstone-project/assets/107192076/0879e989-c981-45e9-9cc3-6c9ccd019aab">



See below for more details:
<table>
<tr>
<th align=“middle”>Behance design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="986" alt="Screenshot 2023-06-01 at 10 52 10" src="https://github.com/Adesina-Abiodun/Capstone-project/assets/107192076/d003d539-e290-4b87-ae6a-ffc413d99cbc">
</td>
<td>
<img width="1440" alt="Screenshot 2023-06-01 at 10 53 01" src="https://github.com/Adesina-Abiodun/Capstone-project/assets/107192076/a0d047d2-17e6-4001-87ca-8a426b4a7c72">
</td>
</tr>
</table>